packages/edit-site/src/components/editor/utils.js

```diff
@@ -29,6 +29,15 @@ export const ROOT_BLOCK_SUPPORTS = [
 	'textDecoration',
 	'textTransform',
 ];
+export const ELEMENTS = {
```

Comment:
Is there a reason to restrict what elements can be used?

Reply:
Not sure what are your thoughts on this (HTML elements?) but the ongoing list of elements planned at the moment is https://github.com/WordPress/gutenberg/issues/27473#issuecomment-796773866 I think is worth starting with a limited set and iterate.

Reply:
My thought was that if I specific it in theme.json it could work regardless of whether we expose that to users.

Reply:
Would you expand a bit? I'm not sure I follow.

Reply:
I want to be able to specify this in my theme.json:
```
"core/latest-comments": {
	"elements": {
		"form": {
			"color": {
				"background": "red"
			}
		}
	}
}
```

At the moment this won't work as `form` isn't a supported element.

Reply:
Right, I get that that's how it works, I'm just wondering why we wouldn't want to expose all HTML elements :)